packages/application-generic/src/services/cron/agenda-cron.service.ts

```diff
@@ -0,0 +1,70 @@
+import { Agenda } from '@hokify/agenda';
+import { MetricsService } from '../metrics';
+import { CronService } from './cron.service';
+import { CronJobProcessor, CronMetrics, CronOptions } from './cron.types';
+
+export class AgendaCronService extends CronService {
+  cronServiceName = 'AgendaCronService';
+
+  constructor(private agenda: Agenda, metricsService: MetricsService) {
```

Comment:
They need to not be private when instantiaing superclass to avoid this error:
<img width="772" alt="image" src="https://github.com/novuhq/novu/assets/32132657/ffa9a5bd-ff14-4db5-b219-4d88b2aab532">